modules/carto/src/layers/vector-tile-layer.ts

```diff
@@ -134,11 +134,19 @@ export default class VectorTileLayer<
     const tileBbox = props.tile.bbox as any;
     const {west, south, east, north} = tileBbox;
 
+    const clipProps = {
+      extensions: [new ClipExtension(), ...(props.extensions || [])],
```

Comment:
Might this prevent other extensions in `props.extensions` from propagating to point layers?